frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleSection.tsx

```diff
@@ -84,6 +88,10 @@ export const allDays = [
 
 const getDayIndexMondaySunday = (date: Date) => (date.getDay() === 0 ? 6 : date.getDay() - 1)
 
+const AlignedTextButton = styled(StyledButton)`
```

Comment:
There already exist a style called "TextAlignedButton" in "StyledComponents.tsx" that is quite similar. Would it give the same result to just import that one instead of defining a new style?

Reply:
Yes, fixed